spatio_temporal_voxel_layer/CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ find_package(visualization_msgs REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 find_package(std_srvs REQUIRED)
+find_package(openvdb_vendor REQUIRED)
```

Comment:
Should we attempt to be a bit more generic here? I'd add a ON/OFF flag to let the user decide if to use a potentially existing OpenVDB installation from the system?

I checked a bit the vendor package, and there is this flag [SATISFIED](https://github.com/ament/ament_cmake/blob/a6a7e0c775545e6f13d767b6012b72ec358248c6/ament_cmake_vendor_package/cmake/ament_vendor.cmake#L20) which sounds that is related for this, but It's not clear if would help. But we could at least warn the user, since OpenVDB takes quite a lof of time to build...

Reply:
I would keep it simple. For releases where openvdb cannot be used, we use openvdb_vendor. Making things optional in the package.xml can be done but is not really common practice.

The SATISFIED syntax is nice. But I would leave that for future enhancement until someone figures out how that works. Typically uses don't compile this anyhow but do an apt-install.

Reply:
fair enough, most users will consume the release package